src/ansible/roles/selenium/files/idp_login_keycloak.py

```diff
@@ -37,11 +43,21 @@
     element = WebDriverWait(driver, 30).until(
         EC.presence_of_element_located((By.ID, "username")))
     driver.find_element(By.ID, "username").send_keys(username)
-    driver.find_element(By.ID, "password").send_keys(password)
+    driver.find_element(By.ID, "password").send_keys(password_old)
```

Comment:
I'm not completely sure this will work. What happens if `password` doesn't contain `:::`?

Reply:
When password_old was defined outside the if condition earlier it worked.   I changed this back to use password though and the if that sets password_new also resets password to the first part if ':::' is included. 

Does that make more sense now?

Reply:
Yes, it does